upgrade/migrate.go

```diff
@@ -79,3 +90,43 @@ func migrateYAMLToJSON() error {
 
 	return nil
 }
+
+func migrateNoNameToName() error {
```

Comment:
The latest fix did not resolve this issue. But I am thinking we might not need this functionality after all. Please see https://github.com/dnote-io/cli/issues/15#issuecomment-296428102